packages/tailwindcss/src/utilities.ts

```diff
@@ -135,6 +135,8 @@ function withAlpha(value: string, alpha: string): string {
   let alphaAsNumber = Number(alpha)
   if (!Number.isNaN(alphaAsNumber)) {
     alpha = `${alphaAsNumber * 100}%`
+  } else if (alpha.startsWith('var')) {
```

Comment:
What about just an `else` here instead of explicitly checking for `var`? Would also work with anyone doing weird `calc` stuff or whatever too then.

Reply:
Changed it slightly so that we check whether the alpha value ends with `%`. If it does, we can use it as-is. If it doesn't, then we wrap it in `calc(… * 100%)`. This will cover `var(…)`, existing `calc(…)` functions and so on.